2/ex2.py

```diff
@@ -0,0 +1,10 @@
+# A comment, this is so you can read your program later.
+# Anything after the # is ignored by python.
+
+print "I could have code like this." # and the comment after is ignored
```

Comment:
Tried the docstring example - I see what you mean in it removing the need for having to preface every line with a #.

Comments I believe are also good for ignoring certain lines of code when running the program to see how that particular line of code affects the entire program.